src/main/kotlin/no/nav/familie/ef/sak/økonomi/dto/StatusFraOppdragDTO.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.familie.ef.sak.økonomi.dto
+
+data class StatusFraOppdragDTO(val fagsystem: String,
```

Comment:
Burde ikke denne vært en del av kontrakt? Ser veldig ut for meg som om denne burde være lik for alle brukere av familie-oppdrag.

Reply:
Ja. Har laget en PR i familie-kontrakter, og en ny branch som tar den i bruk